boards/ct/ctcc/doc/index.rst

```diff
@@ -114,84 +172,99 @@ logs on emulated USB port.
 
-Option 2: Using MCUboot with DFU support
-----------------------------------------
+.. tabs::
 
-It is also possible to use the MCUboot bootloader with DFU support to flash
-Zephyr applications. You need to flash MCUboot with DFU support and fill in slot0 with
-some application one-time using Option 1. Then you can re-flash an application using DFU utility
-by loading images to slot1. Note, it's not possible to have only MCUboot and load directly
-software to slot0 due to DFU implementation in Zephyr, which for present slot0 and slot1 in flash
-map, it assumes only slot1 partition as writeable.
+   .. group-tab:: nRF52840
 
-Install ``dfu-util`` first and make sure MCUboot's ``imgtool`` is
-available for signing your binary for MCUboot as described on :ref:`west-sign`.
+      To build the MCUboot:
 
-Next, do the **one-time setup** to flash MCUboot with DFU support.
-We'll assume you've cloned the `MCUboot`_ as a submodule when initializing
-Zephyr repositories using :ref:`west` tool.
+      .. code-block:: console
 
-#. Compile MCUboot as a Zephyr application with DFU support.
+         west build \
+            -b ctcc/nrf52832 \
+            mcuboot/boot/zephyr
 
-   .. zephyr-app-commands::
-      :app: mcuboot/boot/zephyr
-      :board: ctcc/nrf52840
-      :build-dir: mcuboot
-      :goals: build
-      :gen-args: -DCONFIG_BOOT_USB_DFU_WAIT=y
+   .. group-tab:: nRF9161
+
+      To build the MCUboot:
+
+      .. code-block:: console
+
+         west build \
+            -b ctcc/nrf9161 \
+            mcuboot/boot/zephyr
 
 #. Flash it onto the board as described in Option 1.
 
-#. Flash other Zephyr application to fill in slot0 e.g:
+#. Flash other Zephyr application over USB using :ref:`MCUmgr-compatible tool <mcumgr_tools_libraries>` and reset target to boot into the image.
+
+.. tabs::
+
+   .. group-tab:: nRF52840
+
+      Build the blinky example with MCUboot support:
+
+      .. code-block:: console
 
-   .. zephyr-app-commands::
-      :zephyr-app: samples/subsys/usb/dfu
-      :board: ctcc/nrf52840
-      :build-dir: dfu
-      :goals: build
-      :gen-args: -DCONFIG_BOOTLOADER_MCUBOOT=y -DCONFIG_MCUBOOT_SIGNATURE_KEY_FILE=\"path/to/mcuboot/boot/root-rsa-2048.pem\"
+         west build \
+            -b ctcc/nrf52832 \
+            samples/basic/blinky \
+            -- \
+            -DCONFIG_BOOTLOADER_MCUBOOT=y \
+            -DCONFIG_MCUBOOT_SIGNATURE_KEY_FILE=\"path/to/mcuboot/boot/root-rsa-2048.pem\"
 
-You can now flash a Zephyr application to the board using DFU util.
-As an example we'll use the :zephyr:code-sample:`usb-cdc-acm-console` sample.
+   .. group-tab:: nRF9161
+
+      Build the blinky example with MCUboot support:
+
+      .. code-block:: console
+
+         west build \
+            -b ctcc/nrf9161 \
+            samples/basic/blinky \
+            -- \
+            -DCONFIG_BOOTLOADER_MCUBOOT=y \
+            -DCONFIG_MCUBOOT_SIGNATURE_KEY_FILE=\"path/to/mcuboot/boot/root-rsa-2048.pem\"
```

Comment:
Please don't drop the usage of `zephyr-app-commands`, it helps ensure the command is actually well-formed and allows us to globally tweak how west snippets look like whenever needed. Here for example, this would also example automatically check that samples/basic/blinky exists.

This comment applies to other places in this file

Reply:
all right, changed back to `zephyr-app-commands`